docs/notebooks/run_notebooks.sh

```diff
@@ -1,20 +1,19 @@
-# generated by run_all_notebooks.py
-echo "running 14 notebooks"
+#!/bin/bash
+#Collect all notebooks (except the ones in the measurements and unfinished folder) to be executed
+#One or more notebooks from the measurements folder or others can be added manually by copying, uncommenting
+#and adjusting the filename of the following line and appending it to the bottom of this script.
+#echo "jupyter nbconvert --to notebook --execute docs/notebooks/measurements/some_notebook_to_add.ipynb" >>$OUTFILE
 
+echo "Collecting notebooks to run"
+OUTFILE="docs/notebooks/run_selected_notebooks.sh"
+/bin/cat <<EOM >$OUTFILE
+# Generated file, do not edit it or put it in github
```

Comment:
Maybe add the generated file to `.gitignore`

Reply:
I will

Reply:
because we don't use a generated file anymore, .gitignore need not be adjusted anymore.